src/libs/version_gen.awk

```diff
@@ -1,7 +1,7 @@
 #
 # SPDX-License-Identifier: BSD-2-Clause-FreeBSD
 #
-# Copyright (C) 2019 Elektra Initiative.
+# copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```

Comment:
Capital if it is not the doxygen command `@copyright`.

```suggestion
# Copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```
Applies to all instances.